src/components/v5/common/ActionSidebar/partials/forms/SplitPaymentForm/partials/SplitPaymentRecipientsField/hooks.tsx

```diff
@@ -106,14 +110,20 @@ export const useRecipientsFieldTableColumns = ({
               <SplitPaymentPayoutsTotal
                 data={dataRef.current || []}
                 token={token}
+                value={
+                  distributionMethodRef?.current ===
+                    SplitPaymentDistributionType.Equal &&
+                  (amount ? moveDecimal(amount, token.decimals) : 0)
```

Comment:
can we safely pass `0` here? i would guess that 0 doesn't show up if other values are present, maybe we could pass `undefined` juuust to make sure
![image](https://github.com/user-attachments/assets/8680ba4f-d3e2-4c6f-859d-00ac3c3d1f3b)